docs/quick-start/tips-and-tricks.md

```diff
@@ -0,0 +1,75 @@
+---
+id: tips-and-tricks
+title: Tips and tricks
+sidebar_label: Tips and tricks
+---
+
+The following section gives you some quick ways to operate Titan. ``<n>`` means type a number on the keypad. "\<Avo\> + " means hold down the first button.
+
+## Fixture control and recording
+
+-  Select all fixtures: \<AVO\> + \<ALL\> \<ALL\> \<ALL\>
```

Comment:
Maybe also Select All Fixtures with intensity: `<SelectIf><SelectIf>` might be useful?

Reply:
Done